### PR TITLE
More strongly suggest a/aaa and remove cname references

### PR DESCRIPTION
**Pull request by EdwardAngert**
## Summary

**[Domains](https://pantheon.io/docs/dns)** - Updates to DNS information to more strongly discourage using CNAME. [PR 5643](https://github.com/pantheon-systems/documentation/pull/5643)

## To do

- [x] Merge #5645 first

## Effect

The following changes are already committed:

- update DNS screenshot
- remove references to cname as it's not available through Dashboard and not recommended

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5643 | Difference |
| --- | --- | --- | --- |
| Performance Score | `44` | `40` | `-4` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5643` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/20031/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/20031/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: source/content/dns.md [hunk 1]
```diff
@@ -174,39 +173,25 @@ Some DNS providers provide a default `CNAME` record for `www` pointing to `@` (t
...
-Name:   fe1.edge.pantheon.io
-Address: 23.185.0.1
-Name:   fe1.edge.pantheon.io
-Address: 2620:12a:8001::1
-Name:   fe1.edge.pantheon.io
-Address: 2620:12a:8000::1
-```
+### Why does my domain say "Setup Recommended?"
+
+If the Platform detects a CNAME record pointed to Pantheon, the **Status** in **Domains / HTTPS** will show as <span class="glyphicons glyphicons-alert text-warning"></span> **Setup Recommended**. Click **Details** to find the value detected for the CNAME on the line with `Remove this detected record` in the status.
```
Comment (ari-gold) [inline source/content/dns.md:192]: They also might need to add the required A/AAAA records, so how about changing the last sentence to something like:

Click **Details** to find the required values for A and AAAA records to add, and the CNAME detected to remove. Login to your DNS provider to make the recommended changes.

Comment (EdwardAngert) [reply source/content/dns.md]: I think I've resolved this in the latest commit https://github.com/pantheon-systems/documentation/pull/5643/commits/101eaf55de3d691e18630af2735106125113c3be

File: source/content/dns.md [hunk 2]
```diff
@@ -162,39 +159,27 @@ Some DNS providers provide a default `CNAME` record for `www` pointing to `@` (t
...
-Name:   fe1.edge.pantheon.io
-Address: 2620:12a:8001::1
-Name:   fe1.edge.pantheon.io
-Address: 2620:12a:8000::1
-```
+### Why does my domain say "Update Recommended?"
+
+The **Status** in **Domains / HTTPS** will show as <span class="glyphicons glyphicons-alert text-warning"></span> **Update Recommended** when the Platform detects a CNAME record pointed to Pantheon, or when A/AAAA records are not detected.
+
+Click **Details** to find the values required for A and AAAA records to add, or to find the CNAME detected on the line with `Remove this detected record` in the status.
```
Comment (ari-gold) [inline source/content/dns.md:180]: this sentence is a little awkward. did I write it? =)

Comment (EdwardAngert) [reply source/content/dns.md:180]: haha nah, I'll take the blame for anything that reads awkwardly - I added the `, or to find` to help separate the two things users would be looking for
Moving this to copy review for @pantheon-systems/docs-admins to help smooth and merge

File: source/content/dns.md [hunk 3]
```diff
@@ -162,39 +159,27 @@ Some DNS providers provide a default `CNAME` record for `www` pointing to `@` (t
...
 
-In the example above, we can create an `A` record for `www` set to `23.185.0.1` to substitute the `CNAME` record.
+[Log in to your DNS provider](/dns-providers) to make the recommended changes.
```
Comment (alexfornuto) [inline source/content/dns.md:182]: This link needs more context:

```suggestion
Log in to your DNS provider to make the recommended changes. We have instructions for [many popular DNS providers](/dns-providers) to help make the required adjustment.
```

File: source/content/dns.md (hunk 2 above)
Comment (alexfornuto) [reply source/content/dns.md:180]: It may be a bit awkward, but it's the best kind of correct and a better phrasing doesn't come to my mind.

**Pull request merged**